plugin/helper.js

```diff
@@ -25,18 +25,7 @@ export const transformRelativeToRootPath = (
 
     let sourcePath = sourceFile.substring(0, sourceFile.lastIndexOf('/'));
 
-    // if the path is an absolute path (webpack sends '/Users/foo/bar/baz.js' here)
-    if (
-      sourcePath.indexOf('/') === 0 ||
-      sourcePath.indexOf(':/') === 1 ||
-      sourcePath.indexOf(':\\') === 1
-    ) {
-      sourcePath = sourcePath.substring(root.length + 1);
```

Comment:
Not sure what was the reasoning behind this, I think `path.resolve()` should work fine for absolute paths regardless. 
This change would turn the `sourcePath` into a relative path and the `path.resolve()` would not take into account the root directory.